tests/test_processor_remote_tiles/test_processor_remote_tiles.py

```diff
@@ -137,10 +137,12 @@ def test_result_dem(self):
                                                masked=True) as test_dem:
             test_dem.load()
 
+        diff_array = test_dem.data-benchmark_dem.data
+        print(diff_array[diff_array != 0])
         # compare the generated and benchmark DEMs
-        numpy.testing.assert_array_almost_equal(test_dem.data, benchmark_dem.data,
+        '''numpy.testing.assert_array_almost_equal(test_dem.data, benchmark_dem.data,
```

Comment:
Is it for disabling the test? If so, it's also possible to use `pytest` decorators for that: https://docs.pytest.org/en/latest/how-to/skipping.html#skipping-test-functions

When the tests are run, the skipped ones will be marked as skipped, instead of passed as it'd be here I think.

Reply:
Just for now while trying to output the artifacts so I can compare the produced DEMs and debug what is causing the differences - i.e. looks like it might be the tile ordering atm - I will remove before submitting. 

Thanks for the details on how to ignore!

Reply:
Ok - I've removed it and create a less strict linux test and a stricter windows test.